product_supplierinfo_import/models/product_supplierinfo_import_template.py

```diff
@@ -8,13 +8,30 @@ class ProductSupplierInfoImportTemplate(models.Model):
     _description = "Templates to be detected when importing vendor pricelist"
 
     name = fields.Char(required=True)
+    supplier_id = fields.Many2one(comodel_name="res.partner")
     header_offset = fields.Integer(required=True, default=0)
-    barcode_header_name = fields.Char(required=True)
+    search_header_name = fields.Char(required=True)
     template_line_ids = fields.One2many(
         comodel_name="product.supplierinfo.import.template.line",
         inverse_name="template_id",
         string="Headers mapping",
     )
+    only_update_existing = fields.Boolean(
+        string="Just Update Existing",
+        help="If enabled, only existing supplierinfo records will be updated. "
+        "New ones will not be created.",
+        default=False,
+    )
+    show_unupdated_rates = fields.Boolean(
```

Comment:
*unupdated* sounds a little bit weird
```suggestion
    show_not_updated_rates = fields.Boolean(